developer/src/kmc/src/commands/build.ts

```diff
@@ -112,22 +113,22 @@ async function build(filename: string, parentCallbacks: NodeCompilerCallbacks, o
     if(fs.statSync(filename).isDirectory()) {
       buildFilename = path.join(buildFilename, path.basename(buildFilename) + KeymanFileTypes.Source.Project);
     }
-    buildFilename = path.relative(process.cwd(), buildFilename).replace(/\\/g, '/');
+    const relativeFilename = path.relative(process.cwd(), buildFilename).replace(/\\/g, '/');
 
     const callbacks = new CompilerFileCallbacks(buildFilename, options, parentCallbacks);
-    callbacks.reportMessage(InfrastructureMessages.Info_BuildingFile({filename:buildFilename}));
+    callbacks.reportMessage(InfrastructureMessages.Info_BuildingFile({filename:buildFilename, relativeFilename}));
```

Comment:
This fixes an issue where certain messages only had a base filename rather than full filename.